lib/mutant/env/bootstrap.rb

```diff
@@ -95,7 +95,7 @@ def scope_name(scope)
       # @api private
       def infect
         config.includes.each(&$LOAD_PATH.method(:<<))
-        config.requires.each(&method(:require))
+        config.requires.each(&Kernel.method(:require))
```

Comment:
Another maybe more useful instance of the same pattern is available here: https://github.com/mbj/mutant/blob/master/bin/mutant#L13-L29

This is the call side for mutants self-zombification. It explicitly passes kernel into the zombifier API, for testing the zombifier I could implement a "fake Kernel" (that also does some message expectations, but not via rspec) to specify the behavior of Zombification.

Here is the test support class: https://github.com/mbj/mutant/blob/master/spec/support/ruby_vm.rb

This example is an instance of the same idea: When multiple APIs that produce the same effect exist (`#require` vs `Kernel.require`) use the one that is easier to unit test.